File: TradingEAC-Java-StockService/src/main/java/tradingeac/java/stockservice/models/StockSymbol.java
package tradingeac.java.stockservice.models;

public class StockSymbol {
    private String symbol;
    public void setSymbol(String value){
        symbol = value;
    }
    public String getSymbol(){
        return symbol;
    }

    private String currency;
    public void setCurrency(String value){
        currency = value;
    }
    public String getCurrency(){
        return currency;
    }

    private String description;
    public void setDescription(String value){
        description = value;
    }
    public String getDescription(){
        return description;
    }

    private String displaySymbol;
    public void setDisplaySymbol(String value){
        displaySymbol = value;
    }
    public String getDisplaySymbol(){
        return displaySymbol;
    }

    private String figi;
    public void setFigi(String value){
        figi = value;
    }
    public String getFigi(){
        return figi;
    }

    private String mic;    
    public void setMic(String value){
        mic = value;
    }
    public String getMic(){
        return mic;
    }

    private String type;
    public void setType(String value){
        type = value;
    }
    public String getType(){
        return type;
    }
}

File: TradingEAC-DotNet-StockService/Models/StockSymbol.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TradingEAC.DotNet.StockService.Models
{
    public class StockSymbol
    {
        public string currency { get; set; }
        public string description { get; set; }
        public string displaySymbol { get; set; }
        public string figi { get; set; }
        public string mic { get; set; }
        public string symbol { get; set; }
        public string type { get; set; }
    }
}
